src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1131,9 +1132,9 @@ public void setShutdownRunnable(Runnable shutdownRunnable)
      * to include. If not specified, all of the conference's endpoints will be
      * included.
      */
-    public JSONObject getDebugState(String conferenceId, String endpointId)
+    public OrderedJsonObject getDebugState(String conferenceId, String endpointId)
```

Comment:
Is there a reason to change the signature too? Looks like the return type can remain the more abstract one.